### DISCUSSION TITLE
Winter Green/Garden waste bin hiatus breaking automations

### DISCUSSION THREAD

**Discussion by Jim-Noble** [robbrad/UKBinCollectionData#1071] [Q&A]
How are folks handling their Council's green waste service going on hiatus over winter? 

Dacorum, for example, drop any mention of the green bin in the results, and as a result the associated entities become available. Obviously this is fairly easy to handle in Home Assistant, but I wonder if there would be value in requesting an enhancement to handle it more gracefully in the integration...?

Example output during winter (possibly the green waste date is still displayed - usually collected alongside the blue bin - but the "Green bin" text is not present...)

```
Bin collection schedule for [redacted] HP3 8BD .

Your usual collection day is Thursday
Next collection on:	Thu, 12 Dec 2024
Blue bin and kerbside caddy
Next collection on:	Thu, 12 Dec 2024
Next collection on:	Thu, 19 Dec 2024
Grey bin and kerbside caddy
Next collection on:	Thu, 19 Dec 2024
```

**Comment by dp247** [comment]
Realistically, we handle it however the council present it - most of the time it gets skipped because the data no longer appears on the page

**Reply by Jim-Noble** [reply]
If councils were good at this stuff, you wouldn't need to be screen scraping the data I guess :-) 
For anyone else's benefit who ends up here, I've wrapped my automations with checks for the entities not being unavailable, or unknown, to keep things working...

**Reply by dp247** [reply]
Too true - can only think of one council of the top of my head that even has an API for this lol
